### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by WizPip** [mattyhempstead/yt-comment-sub-count#3]
As of some days ago the plugin is now showing incorrect data (mostly **l>**) (Tested in Windows Chrome only)

**Comment by zloboslav** [comment]
Same issue on Firefox 125.0.3 (64-bit), Windows 10.
It shows mostly "l>" or "Private".
Once every dozen or so there may be a value, but it is always very wrong.

**Comment by mattyhempstead** [comment]
Thanks for letting me know!

A fix has been deployed in version 1.1.9.

**Comment by MarioLiebisch** [comment]
Was there another change breaking this again? Just noticed I see "Private" for everyone, even though I double-checked and their subscriber counts are visible.

**Comment by mattyhempstead** [comment]
@MarioLiebisch It works for me rn on Chrome.

Are you using Firefox? I just realised I didn't deploy the v1.1.9 fix on Firefox so I have done that now. If you are definitely using v1.1.9 then could you open a new issue with more details so I can try to reproduce.

**Comment by MarioLiebisch** [comment]
Nope, I'm on latest Chrome and the extension shows as 1.1.9, too.

However, I just noticed I get it working again, if I switch the website language to English (set to German by default). I had it on English in the past, that might have been the reason why it worked before (didn't check revision history right now).

I just checked the source code and you're indeed looking for "subscribers" as part of the regular expression. That explains it, I guess.
